### ISSUE TITLE
Error when passing --prefix option to `setup.py build`

### ISSUE THREAD

**Issue by brewlius-cesar** [pyne/pyne#651]
Perhaps not necesarrily a bug, but when attempting to perform just a build (not an install) using the --prefix option the command stops with the message 

```
python setup.py --prefix=/home/justin/git/pyne_build build

[Lots of Build Output Here...]

[100%] Built target models

--------------------

If you are having issues building pyne, please report your problem to pyne-dev@googlegroups.com or look for help at http://pyne.io

--------------------
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --prefix not recognized
```

The build actually does finish, the message is just misleading to a newbie like me.  The cause appears to be distutils.core.setup is assuming argv[1:] are the script_options if no script_options are passed in.  It seems like the build step still requires prefix if it is to be used, as this gets passed on to -DCMAKE_INSTALL_PREFIX, etc.

Let me know if this is something you care about and I'd be happy to propose a patch.  (Probably just pass an empty script_options.)

Also, I am just getting started using PyNE but the tools seems feature rich, Thanks!

**Comment by crbates** [comment]
`--prefix=` goes after the build/install command like:
```sh
python setup.py build --prefix=/home/justin/
```
let us know if this doesn't work. We use a somewhat complicated process to build PyNE cmake and `distutils` strapped together so sometimes weird stuff happens.

**Comment by brewlius-cesar** [comment]
Still getting the same message.  I realized today that I was using Python 3.4.2 which it doesn't seem like you are intending to support.  So I retried again today with Python 2.7.9 and using your advice from above, but the error still shows up.

**Comment by crbates** [comment]
If you post the full output either here or in a gist that would be helpful. We test on 3.4.2 every night though PyNE's mesh features are not available on Python 3 yet.

**Comment by crbates** [comment]
This issue is arising because `distutils` does not support setting `prefix` during build.

**Comment by scopatz** [comment]
Ahh we should do a quick refactor to optionally use setuptools if available, and fallback to distutils as needed. I thought it was like that already....

**Comment by scopatz** [comment]
I have opened #652 to address this issue. Thanks for reporting @brewlius-cesar. Also, I run exclusively Python 3 these days, so we definitely support it :)

**Comment by brewlius-cesar** [comment]
Not a problem. Do you still need the full output?

**Comment by crbates** [comment]
No this is a fundamental limitation of `distutils` so the rest of the output won't tell us anything. Thanks.

**Comment by scopatz** [comment]
Moving this to v0.6

**Comment by crbates** [comment]
closed by #687/ #652
